test/unit/testDeceasedMarried.js

```diff
@@ -14,23 +14,37 @@ describe('DeceasedMarried', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
```

Comment:
Please update the description for this test as it is for deceased married but it is referring to deceased address.